start.sh

```
@@ -0,0 +1,3 @@
+#!bin/sh
```

Shellcheck (0.10.0)

[error] Ensure the shebang uses an absolute path to the interpreter.

(SC2239)